util.py

```diff
@@ -1967,6 +1967,70 @@ def parse_mf2(input, url=None, id=None):
   return mf2py.parse(url=url, doc=input, img_with_alt=True)
 
 
+def clean_leading_and_trailing_quotes(url):
+  """Cleans a url by removing leading/trailing quotes at ends if present.
+  
+  Adding due to poor guidance on https://www.w3.org/TR/WCAG20-TECHS/H76.html 
+  where examples show wrapping `anyUrl` with single quotes while procedures
+  show wrapping `URL=anyUrl` with single quotes. This discrepancy is minor and
+  can be easily solved by removing and first and last characters if single
+  quotes. 
+
+
+  Args:
+    url: str
+
+  Returns: str
+  """
+  if url.endswith('\''):
+    url = url[:-1]
+  
+  if url.startswith('\''):
+    url = url[1:]
+  
+  return url
+
+
+def parse_http_equiv(content):
+  """Parses the value in the http_equiv meta field and returns the url.
+
+  Args:
+    content: str, http_equiv content string https://www.w3.org/TR/WCAG20-TECHS/H76.html#procedure
+
+  Returns: str, empty if content format is incorrect
+  """
+  split = content.rpartition('URL=')
+  if not split[1]: # If URL= is not in the string return an empty string
+    return ''
+
+  return clean_leading_and_trailing_quotes(split[2])
```

Comment:
Argh I think one of my comments got lost: there's a builtin for this, you can replace this with `split[2].strip("'")` and drop the function. The comment in its docstring is great though, feel free to move it here!

Reply:
i could use this for sure. i avoided it, because when i looked up the apostrophe i found that it was reserved for URLs per (RFC 3986)[https://datatracker.ietf.org/doc/html/rfc3986] and thought maybe they are used for a specific urn. that is probably sa stretch though.

change has been made!